.github/workflows/labeler.yml

```diff
@@ -0,0 +1,11 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v3
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I guess this is "root token"? Not sure if there's anything better we can use (i.e. a scoped token of some kind)

Reply:
We can use an equivalent here by creating specific token based on permissions and adding it as a secret in the repo settings. But not really sure how to do that.